examples/kubernetes/demo-vol-update.yml

```diff
@@ -0,0 +1,99 @@
+apiVersion: storage.k8s.io/v1alpha1
```

Comment:
Can we change all of these to v1beta1 now that the feature has been promoted to beta?